build_debian.sh

```diff
@@ -317,9 +326,21 @@ sudo LANG=C chroot $FILESYSTEM_ROOT useradd -G sudo,docker $USERNAME -c "$DEFAUL
 ## Create password for the default user
 echo "$USERNAME:$PASSWORD" | sudo LANG=C chroot $FILESYSTEM_ROOT chpasswd
 
-## Create redis group
-sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f redis
-sudo LANG=C chroot $FILESYSTEM_ROOT usermod -aG redis $USERNAME
+## Create remapped redis group and user, add default user to remapped redis group
+sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f -g $REDIS_REMAPPED_GID $REMAPREDIS
+sudo LANG=C chroot $FILESYSTEM_ROOT useradd -u $REDIS_REMAPPED_UID -g $REMAPREDIS $REMAPREDIS -c "remapped docker redis user" -m -s /bin/bash
+sudo LANG=C chroot $FILESYSTEM_ROOT usermod -aG $REMAPREDIS $USERNAME
+
+## Check expected entry in /etc/subuid and /etc/subgid
```

Comment:
>Check [](http://example.com/codeflow?start=3&length=5)

Since you are checking for expectation, do you try to prevent some error situations? If it really happens and we see the error message, what should we do to resolve it?